$MirOS: ports/print/ghostscript/esp/patches/patch-pstoraster_cups_mak,v 1.1.7.1 2005/03/18 15:51:03 tg Exp $
--- pstoraster/cups.mak.orig	Mon Feb 21 16:16:38 2005
+++ pstoraster/cups.mak	Thu Apr 27 16:09:00 2006
@@ -24,9 +24,9 @@
 
 cups_=	$(GLOBJ)gdevcups.$(OBJ)
 
-CUPSSERVER= $(install_prefix)`cups-config --serverbin`
-CUPSCONFIG= $(install_prefix)`cups-config --serverroot`
-CUPSDATA= $(install_prefix)`cups-config --datadir`
+CUPSSERVER= $(DESTDIR)`cups-config --serverbin`
+CUPSCONFIG= $(DESTDIR)`cups-config --serverroot`
+CUPSDATA= $(DESTDIR)`cups-config --datadir`
 
 $(DD)cups.dev:	$(cups_) $(GLD)page.dev
 	$(ADDMOD) $(DD)cups -lib cupsimage -lib cups
